scipy/stats/_qmc.py

```diff
@@ -1739,8 +1724,25 @@ def reset(self) -> MultivariateNormalQMC:
             Engine reset to its base state.
 
         """
-        super().reset()
         self.engine.reset()
+        self.num_generated = 0
+        return self
+
+    def fast_forward(self, n: IntNumber) -> QMCEngine:
+        """Fast-forward the sequence by `n` positions.
```

Comment:
I don't think we need convenience functions like this, but if we do, we can re-use code by establishing a base class `QMCDistribution`.